### DISCUSSION TITLE
en-us/manual/introduction/integrated-strategy

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#175] [Comments]
# en-us/manual/introduction/integrated-strategy

Integrated Strategy (I.S.) This page may be outdated. MAA selects the latest theme by default, and can be changed in Auto I.S. - General. Warning All features involving auto-bat...

https://docs.maa.plus/en-us/manual/introduction/integrated-strategy.html

**Comment by Flavescent** [comment]
Thank you! Is there a way to configure MAA to use the Expeditioner Record mode, rather than the Regular Investigations mode?
